app/api/v1/routes/health.py

```
@@ -1,33 +1,58 @@
-		##TODO add logging
-		return JSONResponse(status_code=503, content={'status': 'error'})
+			raise HTTPException(
+				status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+				detail={
+					'status': 'error',
+					'service': 'redis',
+					'message': 'Redis health check failed',
+				},
+			)
+		return StandardResponse(message='Redis connection is healthy', payload={'status': 'ok'})
+	except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)